content/applications/websites/website/pages/menus.rst

```diff
@@ -0,0 +1,87 @@
+=====
+Menus
+=====
+
+Menus are used to organize your website’s content effectively and help visitors navigate through
+your web pages effectively. User-friendly and well-structured website menus also play a crucial role
+in improving :doc:`search engine rankings <seo>`.
+
+Odoo allows you to customize the content and appearance of these menus to your needs.
+
+Menu editor
+===========
+
+To edit your website's menu, go to :menuselection:`Website --> Site --> Menu Editor`. From there,
+you can:
+
+- rename a menu item or change its URL using the :guilabel:`Edit Menu Item` icon;
+- delete a menu item using the :guilabel:`Delete Menu Item` icon;
+- move a menu item by drag and dropping it to the desired place in the menu;
+- create a drop-down menu by drag and dropping the sub-menu items to the right, underneath their
+  parent menu.
+
+.. image:: menus/menu-editor.png
+   :alt: Menu editor with sub-menus
+
+.. note::
+  - By default, pages are added to the menu when :doc:`they are created <../pages>`.
+  - You can also access the menu editor by clicking :guilabel:`Edit`, then selecting any menu item
+    and clicking the :guilabel:`Edit menu` icon.
+
+Creating menu items
+-------------------
+
+To create a menu item, click :guilabel:`Add Menu Item`, then, in the pop-up window that appears on
+the screen, enter the :guilabel:`Name` and :guilabel:`Url` of the related page. Click :guilabel:`OK`
```

Comment:
```suggestion
the screen, enter the :guilabel:`Name` and :guilabel:`Url or Email` of the related page. Click :guilabel:`OK`
```

Reply:
I know what you mean here since this is the actual label of the field but I think it doesn't make much sense to mention "email" in the context of the sentence. Would removing the guilabel solve this issue here?